subsystem/cluster_test.go

```diff
@@ -2219,6 +2221,59 @@ var _ = Describe("cluster install", func() {
 				}
 			})
 
+			It("reset cluster with manifests", func() {
```

Comment:
Can this say what the test expects? Maybe `reset cluster doesn't delete manifests`?

Reply:
fixed.